saleor/core/views.py

```diff
@@ -24,3 +21,13 @@ def home(request):
 @staff_member_required
 def styleguide(request):
     return TemplateResponse(request, 'styleguide.html')
+
+
+def impersonate_with_msg(request, uid):
```

Comment:
When overriding things it's good to keep the original naming. I think this view could be simply named `impersonate`. Then to avoid the name clashing you could import the original one as `orig_impersonate`. Also, the URL should be named `impersonate-start`.

Reply:
ACK.